src/components/UpdateEmailCheckEmail.vue

```diff
@@ -0,0 +1,49 @@
+<script setup lang="ts">
+import { getAuth, verifyBeforeUpdateEmail } from 'firebase/auth';
+import { onBeforeMount, ref } from 'vue';
+
+import { router } from '../router';
+
+const auth = getAuth();
+
+const props = defineProps(['newEmail']);
+
+const currentEmail = ref();
+
+async function resendEmail() {
+  try {
+    await verifyBeforeUpdateEmail(auth.currentUser!, props.newEmail);
+  } catch (error) {
+    // TODO: Show error in UI.
+    console.log(error);
+    return;
+  }
+  alert('Update Email Sent!');
```

Comment:
Did you mean to check this in?

Reply:
Yes, currently there's no proper indication in the UI that the email was resent.

Reply:
Can we show it in the UI instead? Maybe under the resend button as a disclaimer